source/jormungandr/jormungandr/scenarios/experimental.py

```diff
@@ -243,9 +269,10 @@ def call_kraken(self, request_type, request, instance, krakens_call):
         """
         logger = logging.getLogger(__name__)
         logger.debug('datetime: %s', request['datetime'])
-        crow_fly_stop_points = set()
+        crow_fly_stop_points = dict()
```

Comment:
Please add a comment explaining the value of this dict. That's not straightforward.